interfaces/cython/cantera/examples/reactors/reactor3.py

```diff
@@ -0,0 +1,61 @@
+"""
+Constant-pressure, adiabatic kinetics simulation.
+Simulation is equivalent to reactor1.py, except for an automatically 
+reduced step size when large changes of temperature are anticipated.
+"""
```

Comment:
I would support using the new method in the existing example(s), rather than having the first examples that the users encounter showcase suboptimal ways of using Cantera.

Reply:
Will certainly do this when the opinions on the PR converge: I left it separate for the time being.

Reply:
merged files.